hypervisor/qemu/network.go

```diff
@@ -33,6 +34,15 @@ func GetTapFd(device, bridge, options string) (int, error) {
 		errno syscall.Errno
 	)
 
+	// check if tapname exists, if true, use existing one instead of create new one
+	if device != "" {
```

Comment:
device should not be null, and even this device is existed on host, it should be added to bridge too.

btw, use TUNSETIFF of ioctl on tun fd with existed device name will attach the fd to the tap device,
so I think we don't need this change.

Reply:
I'll change it back.

Because I don't know how to use TUNSETIFF, I choose a more direct way.  But it's ok to me to remove this part of logic.